src/main/java/mate/academy/internetshop/service/BucketService.java

```diff
@@ -1,8 +1,21 @@
 package mate.academy.internetshop.service;
 
+import java.util.List;
 import mate.academy.internetshop.model.Bucket;
 
 public interface BucketService {
 
-    Bucket addItem(Long BucketId, Long ItemId);
+    Bucket create(Bucket bucket);
+
+    Bucket get(Long id);
+
+    Bucket update(Long id);
+
+    Bucket delete(Long id);
+
+    Bucket addItem(Long bucketId, Long itemId);
+
+    Bucket clear(Long bucketId);
+    
+    List getAllItems(Long bucketId);
```

Comment:
```suggestion
    List<Item> getAllItems(Long bucketId);
```
the same in all methods